src/psyclone/transformations.py

```diff
@@ -435,8 +424,7 @@ def _directive(self, parent, children, collapse=None):
                 "directives.")
 
         from psyclone.psyGen import OMPDoDirective
```

Comment:
Please could you move this import to the top (if possible)

Reply:
Done